doc/sections/structure.md

```diff
@@ -13,6 +13,17 @@ Structure of directories and files
    + `vim/`: code for vim
    + `vscode/`: code for vscode
 
+     - `.vscode/*.json`: config for launching, debugging, building the extension
+     - `src/*.ts`: the VSCode plugin source code written in the TypeScript language
+     - `package.json`: the manifest of the plugin (activation events, scripts, dependencies, ...)
+     - `tsconfig.json`: TypeScript configuration (directories, ...)
+     - `vscode.proposed.d.ts`: unstable VSCode API (likely to change)
```

Comment:
no need to say "unstable" and "(likely to change)"

Reply:
it's not "unstable and likely to change" because of us ... it's in Microsoft side : they say that this API is unstable and likely to change

Reply:
I see. Perhaps you should mention here that this file is coming from Microsoft.